src/trace/tracemap.ts

```diff
@@ -207,7 +210,13 @@ export default class TraceMap {
   //   }
   // }
 
-  async trace (specifier: string, parentUrl = this.mapBase, env = ['import', ...this.env]): Promise<string> {
+  /**
+   * @returns `resolved` - either a URL `string` pointing to the module or `null` if the specifier should be ignored.
+   */
+  async trace (specifier: string, parentUrl = this.mapBase, env = ['import', ...this.env]): Promise<string | null> {
+    // TODO: support ignoring prefixes?
```

Comment:
I think it would be nice to support `react/` type boundaries yeah. Possibly even scoped ignores as well so similar structures to the import map itself.